packages/cli/src/generators/app.ts

```diff
@@ -45,21 +45,14 @@ class AppGenerator extends Generator<AppGeneratorOptions> {
   async postWrite() {
     const pkgJsonLocation = join(this.destinationPath(), 'package.json')
     const pkg = readJSONSync(pkgJsonLocation)
-    const pkgDependencies = Object.keys(pkg.dependencies)
-    const pkgDevDependencies = Object.keys(pkg.devDependencies)
 
     console.log('') // New line needed
     const spinner = log.spinner(log.withBranded('Retrieving the freshest of dependencies')).start()
 
-    const dependenciesArray = await Promise.all([
-      replaceDependencies(pkg, pkgDependencies, 'dependencies'),
-      replaceDependencies(pkg, pkgDevDependencies, 'devDependencies'),
-    ])
+    pkg.dependencies = await fetchLatestVersionsFor(pkg.dependencies);
+    pkg.devDependencies = await fetchLatestVersionsFor(pkg.devDependencies);
```

Comment:
Also, is there a reason you removed the Promise.all?

Reply:
I thought it'd be easier to read - but a day later, I don't think that's necessary true. I'll change it to use Promise.all again ^^